File: client/di_core/domain/Response/Authentication/RegisterResponse.ts
import { UserInfo, UserProfile } from '@core/domain/Model';

export class RegisterResponse {
  constructor(public userInfo: UserInfo, public userProfile: UserProfile) {}

  static fromObject(obj: any): RegisterResponse {
    return new RegisterResponse(UserInfo.fromObject(UserInfo.fromObject(obj.userInfo)), UserProfile.fromObject(obj.userProfile));
  }
}

File: rocket-bi-server/src/main/scala/co/datainsider/caas/user_profile/domain/profile/RegisterResponse.scala
package co.datainsider.caas.user_profile.domain.profile

import co.datainsider.caas.user_profile.domain.user.{UserInfo, UserProfile}

/**
  * @author anhlt
  */
case class RegisterResponse(userInfo: UserInfo, userProfile: UserProfile)

case class VerifyForgotPasswordResponse(token: String)
